### ISSUE TITLE
Pint is unable to parse its own pretty print

### ISSUE THREAD

**Issue by ClaymoreX** [hgrecco/pint#2060]
When I try to read a value that I've had pint print out in pretty format, I get the following error:

pint.errors.UndefinedUnitError: '×10' is not defined in the unit registry

The value in question is 1/58e6 * ureg.ohm * ureg.meter, which pretty prints to 1.72×10⁻⁸ m·Ω

**Comment by dalito** [comment]
As far as I know it has never been a clear goal to support such round-tripping. So this is more a feature request than a bug report.

Did you test what is causing the problem, the magnitude `1.72×10⁻⁸` or the unit `m·Ω` or both?

**Comment by ClaymoreX** [comment]
It appears to be the x. If I change the x to an * it parses the rest of it just fine.

**Comment by jules-ch** [comment]
I don't think we want to implement this, this complicates things unecesseraly IMO.

Maybe a warning in the docs ?
